README.md

```diff
@@ -94,6 +94,31 @@ If you, for some reason, need to remove the coloring, use:
 cargo modules --plain
 ```
 
+### Dot mode
+
+If you want also want to see which module depends on which other modules you can you use the dot mode to output Graphviz Dot compatible output
+
+```bash
+cargo modules --dot
+```
+
+![dot-preview](dot-preview.png)
+
+As extra options you can toggle external types/modules, conditional modules and used types using the `--external`, `--conditional` and `--types` options respectively
```

Comment:
I think here and in the lines below, you should end sentences with periods for consistency with the rest of the readme.